### DISCUSSION TITLE
FireFly show charts for transactions for all years

### DISCUSSION THREAD

**Discussion by ahmednawazkhan** [firefly-iii/firefly-iii#5516] [Firefly III] [answered]
Hello. I am using FireFly for managing my house construction expenses. It was catering all my needs perfectly. Now that year has changed I am unable to see the category graph for the year 2021. It is only showing me expenses from 2022 ( dashboard graphs ). Is there a way that I tell firefly to ignore the year altogether and show me everything on the dashboard irrespective of year. Thanks

**Comment by JC5** [comment, marked as answer]
You can change the range on the top of the page or use the /reports page.

**Reply by ahmednawazkhan** [reply]
thank u
